Daytime-Operations/Introduction-Combined-Calibrations-Procedure/introduction-combined-calibrations-procedure.rst

```diff
@@ -0,0 +1,76 @@
+.. |author| replace:: *Andrés A. Plazas Malagón*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _Introduction-Combined-Calibrations-Generation-Procedure:
+
+##########################################################
+Introduction to Combined Calibrations Generation Procedure
+##########################################################
+
+.. _Introduction-Combined-Calibrations-Procedure-Overview:
+
+Overview
+========
+
+This procedure describes how to execute the SAL Script to take individual calibration frames (biases, darks, and flats) and verify them via the `cp_verify`_ package for LATISS or LSSTComCam on the ScriptQueue from the LSST Operations and Visualization Enviroment (LOVE) at the Summit. 
+
+The script will have the option to: 
+
+- command the instrument to take a number of individual calibrations frames:
+   - biases
+   - darks
+   - flats
+- call the corresponding Rubin Science Pipelines calibration generation pipetask to produce combined biases, combined darks, combined flats, defects, and Photon Transfer Curves (PTCs) via the OCS-Controlled Pipeline System (OCPS),
+- verify the images taken using as reference the calibration generated in the previous step via the ``generate_calibrations`` boolean parameter in the configuration file (this mode is referred to as "internal verification"; see the package `cp_verify`_, and the technical notes `DMTN-101`_ and `DMTN-222`_), or using pre-existing calibrations that are usually located in the ``LATISS/calib`` or `LSSTComCam/calib` standard calibration collections (this mode is referred to as "external verification"), and
+- automatically certify the newly-generated combined calibration (if ``generate_calibrations`` is set to ``True``) with a given range of validity dates (usually one day).
+
+The verification step is currently only implemented for biases, darks, and flats.
+In order to run `cp_verify`_, the verification pipetask uses as input individual images and the combined image for a particular calibration type as a reference.
+This reference combined image may already exist in the default calibration collection ``LATISS/calib`` or ``LSSTComCam/calib`` (a location that can be changed via the ``input_collections_verify_bias`` and similar parameters), or, the user can indicate the script to generate a new combined calibration from the individual images taken via the ``generate_calibrations`` parameter.
+The former is known as ``external verification``, and the latter as ``internal verification``.
```

Comment:
This sentence no longer works here because you don't discuss the two just above.

Reply:
The two options are mentioned in the sentence immediately before: 

"This reference combined image may already exist in the default calibration collection ``LATISS/calib`` or ``LSSTComCam/calib`` (a location that can be changed via the ``input_collections_verify_bias`` and similar parameters), or, the user can indicate the script to generate a new combined calibration from the individual images taken via the ``generate_calibrations`` parameter. "

Reply:
"If you don't build the calibrations, when thy do verification tasks still run?" 

Because that would be when we do "external verification", which is currently the default.